omero/developers/testing.txt

```diff
@@ -86,7 +90,7 @@ explicitly via:
 Results are placed in ``components/<component>/target/reports``.
 
 Individual test groups
```

Comment:
I wonder, given that this is only available to Java tests, whether it should be in the next section? I.e. reserve the current section for things that can be done in broadly similar ways with any component?